products/statement-generator/src/styles/utilityStyles.ts

```diff
@@ -47,6 +47,7 @@ const useUtilityStyles = makeStyles<Theme>(
         display: 'flex',
         flexDirection: 'row',
         width: '100%',
+        justifyContent: 'flex-end',
```

Comment:
@ramankala Wouldn't this impact every `buttonContainer` layout? I think it's have same effect on the `Start Now` button on the home page.
If that's what's intended then cool, but the issue seems to only call out the `let's continue` button.

Reply:
@tglaz Thanks for catching this! I just made the changes so that the `Let's Continue` button is only impacted by the change, and not the` Start Now` button on the home page.